README.md

```diff
@@ -4,6 +4,10 @@ A frontend for creating and managing email subscriptions.
 
 ## Overview
 
+### Running the application
+
+This application is designed to be run within the [GOV.UK Docker](https://github.com/alphagov/govuk-docker) development environment it will be available on the host at http://email-alert-frontend.dev.gov.uk/.  Using the `--live` flag can be used to test a read-only frontend app against live GOV.UK APIs.
```

Comment:
I think some clarification around the `--live` flag is needed here as this flag is only available via the [startup script] rather than when using Docker. However, it does look like docker supports running the app in [live mode] too via `govuk-docker-up app-live`. Maybe we could update this last sentence so it just relates to Docker?

[startup script]: https://github.com/alphagov/email-alert-frontend/blob/master/startup.sh
[live mode]: https://github.com/alphagov/govuk-docker/blob/master/projects/email-alert-frontend/docker-compose.yml#L47-L59

Reply:
Sure, what would you like the ideal copy to read? I can update.

Reply:
Possibly something like:

```
You can run Email Alert Frontend against live, read-only data with `govuk-docker-up app-live`.
```

Reply:
Ace, that's been updated. Hopefully that reads better?

Reply:
Yup, looks good to me! Thanks for updating.

Reply:
Np, thanks for taking a gander and offering the ideal copy. 🙌🏼

Reply:
Ahh I've realised I don't have auth on this one - would you be ok to the honours and click merge? @1pretz1

Reply:
Ah, probably because Email Alert Frontend is continuously deployed and uses stricter merge permissions.

Reply:
Sure thing, I like pressing big green buttons 😄